Tasks/AzureWebPowerShellDeployment/task.loc.json

```diff
@@ -13,7 +13,7 @@
   "version": {
     "Major": 1,
     "Minor": 0,
-    "Patch": 34
+    "Patch": 35
```

Comment:
Just anything using VstsAzureHelpers_ was my concern. Only common modules explicitly referenced get copied into the task -  see common.json at the root of the repo.